src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+@import 'var';
+
+.card {
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  background: $background-white-color;
+  max-width: 198px;
+  margin-bottom: 500px !important;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+
+    & img {
+      width: 100%;
+    }
+  }
+
+  &__title {
+    color: rgb(6, 11, 53);
+    font-size: $text-size;
+    font-weight: 500;
+    line-height: 17px;
+    padding-top: 40px;
+  }
+
+  &__code {
+    color: rgb(97, 96, 112);
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    padding-top: 4px;
+  }
+
+  &__review {
+    padding-top: 16px;
+    display: flex;
+    align-items: center;
+    margin-bottom: 24px;
+    justify-content: space-between;
+    &-text {
+      font-size: 10px;
+      width: 53px;
+      text-align: right;
+    }
+  }
+
+  &__price {
+    color: $price-color;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    letter-spacing: 0%;
+    text-align: right;
+
+    // &-text {
+    //   color: rgb(97, 96, 112);
+    //   font-size: $text-size;
+    //   font-weight: 400;
+    //   line-height: 18px;
+    //   letter-spacing: 0%;
+    //   text-align: left;
+    // }
+
+    &-text {
+      color: rgb(97, 96, 112);
+      font-size: 12px;
+      line-height: 18px;
+      letter-spacing: 0%;
+    }
+  }
+
+  &__value {
+    justify-content: space-between;
+    margin-bottom: 16px;
+    display: flex;
+  }
+
+  &__button {
+    color: #fff;
+    background-color: #00acdc;
```

Comment:
The color `#00acdc` should be defined as an SCSS variable for better maintainability.